src/core.ts

```diff
@@ -73,12 +91,12 @@ export default {
     TransparentNeutral400: 'rgba(203, 211, 223, 0.55)',
     TransparentScrim: 'rgba(0, 0, 0, 0.25)',
     TransparentScrimDarker: 'rgba(0, 0, 0, 0.50)',
-  },
+  } as ColorType,
   Font: {
-    Regular: 'ProximaNovaRegular, Helvetica, Arial, sans-serif',
-    Semibold: 'ProximaNovaSemibold, Helvetica, Arial, sans-serif',
-    Bold: 'ProximaNovaBold, Helvetica, Arial, sans-serif',
-  },
+    Regular: 'Inter, Helvetica, Arial, sans-serif',
+    Semibold: 'Inter, Helvetica, Arial, sans-serif',
+    Bold: 'Inter, Helvetica, Arial, sans-serif',
```

Comment:
If `Semibold` and `Bold` will be the same as `Regular`, should we just remove these as options and allow them to be controlled in `FontWeight`?

Reply:
Long term yes, however, in order to keep this backwards compatible, I think we need to leave it for now.

Reply:
Changed Inter to match the old naming pattern, so now we have `InterRegular`, `InterSemiBold` and `InterBold`